docs/2.8.0/docs/daml-finance/concepts/lifecycling.rst

```diff
@@ -109,6 +114,11 @@ we prevent holders from (accidentally or intentionally) claiming a particular ef
    :alt: The issuer processes the distribution event through the distribution rule, creating a
          lifecycle effect. The effect references ACME v1 as a target instrument.
 
+These components are described in more detail in the Components section below. For simplicity, we
+have chosen a dividend as an example here, but the same workflow can be used to process more
+advanced instruments involving more complex calculations and where the resulting v2 instrument is
```

Comment:
```suggestion
complex events.
```

I would then move the sentence from above "Note
that the logic to create the new version of an instrument can also be encoded in the lifecycle rule." here.

It doesn't make sense where it currently sits, as we haven't introduced lifecycle rules yet.

Reply:
If I commit this suggestion, I think it would read 
_, but the same workflow can be used to process more complex events
automatically defined and produced._

Do you mean the following?
_, but the same workflow can be used to process more
complex events and where the resulting v2 instrument is
automatically defined and produced._

Reply:
I meant to remove the "and where the ..." and replace it with the sentence "Note
that the logic to create the new version of an instrument can also be encoded in the lifecycle rule."